src/software/thunderscope/ProtoReceiver.py

```diff
@@ -0,0 +1,43 @@
+import software.thunderscope.constants as constants
+import queue
+from threading import Thread
+from software.networking.threaded_unix_listener import ThreadedUnixListener
+import software.thunderscope.constants as constants
+
+
+class ProtoReceiver:
+    def __init__(self):
+        self.proto_map = dict()
+        self.proto_receiver = ThreadedUnixListener(
+            constants.UNIX_SOCKET_BASE_PATH + "protobuf",
+            convert_from_any=True,
+            max_buffer_size=3,
+        )
+        self.thread = Thread(target=self.start)
+        self.thread.start()
+
+    """
+    Distributes protobuf from the proto_receiver to all widgets that consume that specific protobuf
+    """
```

Comment:
this should be under the function

~~~
def start(self):
    move comment here
    while True
~~~